packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/SliceZone/SlicesTemplatesModal.tsx

```diff
@@ -18,7 +18,7 @@ import UpdateSliceZoneModalList from "./UpdateSliceZoneModalList";
 interface UpdateSliceModalProps {
   formId: string;
   close: () => void;
-  onSuccess: (slices: SharedSlice[]) => void;
+  onSuccess: (slices: SharedSlice[], libraryName: string) => void;
```

Comment:
i'd use `library` here, and also an object to avoid positional arguments

Reply:
Done ☑️